src/sass/rx/rx.scss

```diff
@@ -90,6 +90,7 @@ button.usa-button-outline {
 
 .rx-loading-error {
   font-weight: bold;
+  width: 66%;
```

Comment:
Great point! Opened #4209 to ask this. In the mean time, I'm going to merge this, as not having the call center number really isn't ideal, so I'm good coming back to the width question to get the call center number in today.